### PR TITLE
Feature/language select styles

### PR DESCRIPTION
**Pull request by mluena**
Footer (first approach) 
- Pages not created for links yet
- Missing Logo, Privacy policy and Terms and conditions

Language select styled

## Testing instructions

Change language in Language Selector (There are translations just for "Select language string si it should be the only one changing")

## Jira

https://vizzuality.atlassian.net/browse/GECHINA-33?atlOrigin=eyJpIjoiNjU5NDEzNmM2ZDQ2NDgwZDlkN2ViZmZlZTU1NWIzMzMiLCJwIjoiaiJ9

### CODE REVIEW

**Review by andresgnlez — CHANGES_REQUESTED** [review]
Conflicts.

File: components/language-select/__test__.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+describe('Translations toggle successfully', () => {
+  it('Translates from EN to ZH CH', () => {
+    cy.intercept({
```
Comment (andresgnlez) [inline components/language-select/__test__.js:3]: Why do you intercept the request?

File: components/language-select/__test__.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      url: '/',
+    });
+    cy.visit('/');
+    cy.get('#downshift-0-toggle-button').click()
+      .invoke('text')
+      .then((text) => {
+        expect(text).to.equal('Select language');
+        cy.get('[data-code="zh_CN"]').click();
```
Comment (andresgnlez) [inline components/language-select/__test__.js:11]: I think we should test that some element, like any text out of this select, is translated properly from one language to another and vice-versa.

File: pages/index.tsx [hunk 3]
```diff
@@ -1,19 +1,16 @@
...
-
-const LanguageSelect = dynamic(() => import('../components/language-select'), { ssr: false });
+import Footer from 'components/footer';
 
 const HomePage: FC = () => (
   <StaticPage className="static-custom">
     <Head title="Welcome to Green Energy Data Platform" />
-    <p className="text-5xl bold">Green energy data platform</p>
-    <LanguageSelect />
+    <Footer />
```
Comment (andresgnlez) [inline pages/index.tsx:13]: Shouldn't the footer go inside the `layout` component? As I think, the footer will be on every page, not exclusively on the homepage.

Comment (mluena) [reply pages/index.tsx:13]: sure but we didn't have it at this point

**Review by andresgnlez — APPROVED** [review]

**Pull request merged**